api-hub/v1beta1/managedclustermodule_types.go

```diff
@@ -38,6 +38,14 @@ type ManagedClusterModuleSpec struct {
 
 // ManagedClusterModuleStatus defines the observed state of ManagedClusterModule.
 type ManagedClusterModuleStatus struct {
+	// Number of ManifestWorks to be applied.
+	NumberDesired int32 `json:"numberDesired"`
```

Comment:
Why does it needs to be `int32` and not just `int`?

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types